src/list-data/request-action-mapping.ts

```diff
@@ -42,17 +25,39 @@ export const BcMapping: RequestActionMappingI = {
   ],
   // Restore
   REH: [
-    EntityTypes.CR,
+    EntityTypes.CR, // maps to BC
     EntityTypes.CP,
     EntityTypes.CC,
-    EntityTypes.UL,
+    EntityTypes.UL, // maps to ULC
     EntityTypes.FI,
-    EntityTypes.BC,
-    EntityTypes.SO
+    EntityTypes.BC, // maps to BEN
+    EntityTypes.SO,
+    EntityTypes.C,
+    EntityTypes.CBEN,
+    EntityTypes.CCC,
+    EntityTypes.CUL
   ],
   // Change Name
-  // (every entity type except Parishes and Private Act)
-  CHG: EntityTypesBC.filter(ent => ent !== EntityTypes.PAR && ent !== EntityTypes.PA),
+  CHG: [
+    EntityTypes.FR,
+    EntityTypes.DBA,
+    EntityTypes.CR,
+    EntityTypes.UL,
+    EntityTypes.GP,
+    EntityTypes.LP,
+    EntityTypes.LL,
+    EntityTypes.CP,
+    EntityTypes.BC,
+    EntityTypes.CC,
+    EntityTypes.SO,
+    EntityTypes.PA,
+    EntityTypes.FI,
+    EntityTypes.PAR,
+    EntityTypes.C,
+    EntityTypes.CBEN,
+    EntityTypes.CCC,
+    EntityTypes.CUL
+  ],
```

Comment:
I'm pretty sure only the REH and CHG arrays are used from here, so those are the only ones I updated.